src/Features/Core/Portable/PullMemberUp/AbstractMembersPullerService.cs

```diff
@@ -265,6 +292,13 @@ private static ISymbol GetSymbolsToPullUp(MemberAnalysisResult analysisResult)
                 options: await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false));
             var newDestination = codeGenerationService.AddMembers(destinationSyntaxNode, pullUpMembersSymbols, options: options, cancellationToken: cancellationToken);
```

Comment:
is there some code somewhere that validates teh destination type's language is the same as the source type's language?

Reply:
It looks like the quick action is not offered when the base type is in another language, as I can't seem to find it when the base class is written in VB and derived in C#. However, I can't find any line of code that validates this ([here is the closest thing](https://github.com/dotnet/roslyn/blob/debd668e68d7db2583e261bfa2563f9acb859bcf/src/Features/Core/Portable/PullMemberUp/MemberAndDestinationValidator.cs#L15)), or any tests that confirm this behavior, so it's possible that there is a check. 

I think the move to base class shouldn't work at all when the languages are different, as the generator takes an option which tells it to reuse the existing member syntax, which should already be in its language-specific form. Moving to an interface should probably still work, since the generator rewrites the member signature from scratch. Would you suggest I add some sort of check here, or would that be another issue entirely?